ert3/data/_record.py

```diff
@@ -18,7 +32,7 @@ def _build_record_index(data):
 
 
 class Record(_DataElement):
-    data: Union[List[float], Mapping[int, float], Mapping[str, float]]
+    data: Union[List[float], Mapping[int, float], Mapping[str, float], List[bytes]]
```

Comment:
Also, would be nice to see both the record and the transmitter tests extended also test for data matching `List[bytes]`..

Reply:
I think we should have a discussion about this, because the current idea to solve the problem in https://github.com/equinor/ert/pull/1469#discussion_r610513414 requires us to introduce a record type, which could also solve this.